### PR TITLE
merge Dss release 70.3 v3 into master

### PR DESCRIPTION
**Pull request by jpotter-DFE**

### CODE REVIEW

File: NCS.DSS.Collections/ServiceBus/DataCollections/Processor/Function/DataCollectionsQueueProcessor.cs [hunk 1]
```diff
@@ -21,8 +19,8 @@ public DataCollectionsQueueProcessor(IDataCollectionsQueueProcessorService dataC
             _loggerHelper = loggerHelper;
         }
 
-        
-        [FunctionName("DataCollectionsQueueProcessor")]
+
+        [Function("DataCollectionsQueueProcessor")]
         public async Task RunAsync([ServiceBusTrigger(_dataCollectionsQueueName,
                                                 Connection = _dataCollectionsConnectionString)]
             MessageCrossLoadToNCSDto message,
```
Comment (hnandra-DFE) [inline NCS.DSS.Collections/ServiceBus/DataCollections/Processor/Function/DataCollectionsQueueProcessor.cs:26]: Why does this function not require removal of the logger from the constructor as per the logger changes we have been making as part of the .NET 8.0 upgrade changes: https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/4574773323/Common+Upgrade+Issues#ILogger-issue

Comment (bmatthews-DFE) [reply NCS.DSS.Collections/ServiceBus/DataCollections/Processor/Function/DataCollectionsQueueProcessor.cs:26]: This has been picked up here: https://github.com/SkillsFundingAgency/dss-collections/pull/31

File: NCS.DSS.Collections/PostCollectionHttpTrigger/Function/PostCollectionHttpTrigger.cs [hunk 2]
```diff
@@ -1,102 +1,89 @@
...
-                collection = await _httpRequestHelper.GetResourceFromRequest<Collection>(req);               
+                _logger.LogInformation("Attempt to get resource from body of the request");
+                collection = await _httpRequestHelper.GetResourceFromRequest<Collection>(req);
             }
             catch (JsonException ex)
             {
-                _loggerHelper.LogError(log, correlationGuid, "Unable to retrieve body from req", ex);
-                return _httpResponseMessageHelper.UnprocessableEntity(ex);
+                _logger.LogError("Unable to retrieve body from req", ex);
+                return new UnprocessableEntityObjectResult(ex.Message);
```
[the remark this answers is not in the document]
Comment (bmatthews-DFE) [reply NCS.DSS.Collections/PostCollectionHttpTrigger/Function/PostCollectionHttpTrigger.cs:86]: To be looked into tomorrow 22/10 - please hook up with me early tomorrow morning
@hnandra-DFE

**Review by bmatthews-DFE — APPROVED** [review]
Approving as this code is already in PRD

**Pull request merged**